### DISCUSSION TITLE
The max tokens of generated answer by ChatGPT is 768？ Is the restricted by proxy or openai?

### DISCUSSION THREAD

**Discussion by jianhai0527** [acheong08/ChatGPT#1033] [Q&A]
RT

**Comment by acheong08** [comment]
For V3, proxy is not used.

**Reply by acheong08** [reply]
All restrictions other than rate limit are from OpenAI for V1

**Reply by jianhai0527** [reply]
got it， I am using V1 and found that the max tokens of generated answers is 768. It seems smaller than Web ChatGPT. Do you have any idea? Thanks a lot.

**Reply by acheong08** [reply]
The max tokens should be 4096 - the number of tokens in prompt

**Reply by jianhai0527** [reply]
yes, I also see that in the api document. However, I got max 768 tokens by V1 method. I am much confused........

**Reply by acheong08** [reply]
I'll check this out

**Reply by acheong08** [reply]
I'm not hitting any token limit. It might be that the model was fine tuned to speak less.

**Reply by acheong08** [reply]
I asked it to count numbers to infinity and it reached the same amount with both manual max_tokens and max_tokens unset
